src/parsers/graph/model-graph.cpp

```diff
@@ -0,0 +1,185 @@
+//
+// Copyright (c) 2025 INRIA
+//
+
+#include "pinocchio/parsers/graph/model-graph.hpp"
+#include "pinocchio/parsers/graph/graph-visitor.hpp"
+
+namespace pinocchio
+{
+  void ModelGraph::addFrame(const std::string & vertex_name, const FrameGraphVariant & frame)
```

Comment:
You can add the `addBody` helper function (with Inertia as parameter instead of the variant)